src/storage/src/hummock/sstable/builder.rs

```diff
@@ -354,7 +384,6 @@ pub(super) mod tests {
     };
 
     #[tokio::test]
-    #[should_panic]
```

Comment:
Why remove this?

Reply:
The test will panic only because original code in SSTBuilder doesn't handle divided-by-zero [here](https://github.com/risingwavelabs/risingwave/blob/main/src/storage/src/hummock/sstable/builder.rs#L308). This PR has fixed this bug.